spec/inquisition/bundler/leak/runner_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Inquisition::Bundler::Leak::Runner do
+  include_examples 'enablable', 'bundler_leak'
+
+  describe '#call' do
+    subject(:runner) { described_class.new }
+
+    let(:scanner) { instance_double(Bundler::Plumber::Scanner) }
```

Comment:
Propose to create next tests
- `when there are no issues`
- `when there is unpatched gem`
- `when there is unsupported issue` (for future purpose)